lib/tasks/users.rake

```diff
@@ -135,6 +135,17 @@ GROUP BY p.user_id
   puts 'Done!', ''
 end
 
+desc "Disable 2 Factor for user."
+task "users:disable_2factor", [:username] => [:environment] do |_, args|
+  username = args[:username]
+
+  user = find_user(username)
+  id = user.id
+  UserSecondFactor.where(user_id: id).destroy_all
```

Comment:
You'll want to do `UserSecondFactor.where(user_id: id).each(&:destroy!)` instead. If something fails to destroy, it'll raise an error and exit.